dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -161,6 +162,16 @@ def authenticate
       if user
         sign_in user
 
+        metadata = {
+          'LMS_Type' => integration.platform_name,
+          'User_Type' => user.user_type,
```

Comment:
Is this the preferred casing/naming convention for Statsig? Do we get any capability to search/index off these metadata fields or just top-level fields?

Reply:
I changed these (and all string keys) to lower snake_case.

Reply:
To answer your question regarding indexing off the metadata fields, from what I can see this isn't supported. Mostly we'll be indexing off of the event name. You can add additional "filters" (they call them "dimensions") on the event value, but I haven't seen anything on being able to index on metadata.